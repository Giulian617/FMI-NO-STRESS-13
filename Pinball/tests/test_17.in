100 10000
14 29 2
58 29 10
48 58 7
92 14 9
35 58 10
4 35 5
81 48 2
65 4 4
80 65 7
60 58 9
94 80 5
100 48 5
9 60 3
34 35 6
99 58 9
91 81 4
33 4 5
84 14 5
67 33 3
50 81 4
71 65 3
96 67 3
89 84 6
97 96 8
68 58 1
90 91 8
17 35 7
75 94 4
63 35 1
87 92 10
98 17 2
88 60 7
44 96 7
69 96 3
43 69 7
83 35 6
95 17 9
72 44 7
55 100 9
8 83 10
61 50 1
6 43 3
32 67 4
56 88 5
77 63 10
47 87 10
54 84 3
73 67 9
21 71 6
40 98 10
46 32 9
45 80 5
86 14 8
11 50 4
25 56 1
15 55 10
53 65 5
74 21 4
10 81 6
36 48 8
70 15 7
66 40 2
18 70 7
82 33 5
5 91 1
51 15 3
85 18 3
23 11 4
49 34 2
28 6 5
52 81 5
24 98 8
64 67 6
93 97 1
38 51 6
16 58 10
76 55 7
26 80 5
3 43 7
57 48 6
13 51 6
42 4 5
19 89 8
7 88 1
39 94 1
78 18 2
30 11 4
1 71 6
31 67 3
37 63 4
12 68 9
59 12 1
20 95 6
2 38 7
22 69 1
79 56 8
27 14 5
41 94 4
62 18 5
