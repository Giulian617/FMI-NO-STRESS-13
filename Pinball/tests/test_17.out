651322716
507950526
777176432
961054959
930981359
418692842
753558080
895709036
507950526
327688575
52911513
931440971
745391832
859553589
398477356
462377208
561759084
889200051
411535163
859553589
535998862
187778869
81729288
506204466
181149356
190577373
561759084
513830949
582459712
418692842
299678897
155963457
90476998
75736100
179829457
718038956
803601291
888322744
339379177
695256403
513563383
101647151
181149356
258635147
918226118
980743987
628127771
166442197
491857331
753864743
595520781
396809020
102663517
